package-lock.json

```diff
@@ -0,0 +1,12685 @@
+{
```

Comment:
Delete this file as well, I think it's comes because you might have used npm to install some package but we should use yarn to add any package.